Project.toml

```diff
@@ -28,6 +29,7 @@ StatisticalMeasures = "a19d573c-0a75-4610-95b3-7071388c7541"
 Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
 TableOperations = "ab02a1b2-a7df-11e8-156e-fb1833f50b87"
 Tables = "bd369af6-aec1-5ad0-b16a-f7cc5008161c"
+ToeplitzMatrices = "c751599d-da0a-543b-9d20-d0a503d91d24"
```

Comment:
As discussed if really needed, this should be in `test/Project.toml`. To add a test dependency you can do:

```julia
]activate test
add ToeplitzMatrices
```

Note that if you want to generate correlated treatments, you don't necessarily need an external package, see for example: https://github.com/TARGENE/TMLE.jl/blob/aeab2c16b0ab3970ac46bd719c823573ea620ac8/examples/interactions_correlated.jl#L89